src/Test/Utilities/Shared/Compilation/OperationTreeVerifier.cs

```diff
@@ -790,7 +807,8 @@ public override void VisitLiteralExpression(ILiteralExpression operation)
         {
             LogString(nameof(ILiteralExpression));
 
-            if (operation.ConstantValue.HasValue && operation.ConstantValue.Value.ToString() == operation.Text)
+            if (operation.ConstantValue.HasValue &&
+                operation.ConstantValue.Value?.ToString() == operation.Text)
```

Comment:
I feel like the appropriate thing to do is just log the constant value.  It's not clear to me why we're looking at the operation text at all.